src/models/kernels/reservoir/Reservoir.cpp

```diff
@@ -105,6 +105,7 @@ double Reservoir::response_meters_per_second(double in_flux_meters_per_second, i
 {
     double outlet_velocity_meters_per_second = 0;
     double sum_of_outlet_velocities_meters_per_second = 0;
+    excess_water_meters = 0;
```

Comment:
I don't see an immediate problem doing this, but the documentation should be very clear that this variable is reset per call.  A caller could choose to hold all excess in the variable and expect an accumulation into it (which we weren't doing.)

Reply:
That typ of usage would have been a little dangerous even before these changes (at least in some situations, any previous value was already going to be lost), but I see your point.  I have pushed another commit with an update to the docstring that expands on how the reference param is handled, and that previous contents will be lost.

Reply:
I think this is fine too.